lib/pagem.rb

```diff
@@ -108,10 +108,13 @@ class PagemMultiscope < Pagem
   def initialize(first_scope, second_scope, params, opt = {})
     super(first_scope, params, opt)
     @first_scope, @second_scope = first_scope, second_scope
-    @first_scope_count, @second_scope_count = first_scope.size, second_scope.size
-    # Add the scope counts together only if count is a number since if otherwise count should be provided
-    # via the count_number optional arg in the inherited class
-    @count = @first_scope_count + @second_scope_count if @count.is_a?(Integer)
+    @first_scope_count = opt[:first_scope_count] || first_scope.size
+    @second_scope_count = opt[:second_scope_count] || second_scope.size
+    if opt[:count_number].blank?
```

Comment:
The <code>initialize</code> in the inherited <code>Pagem</code> module would have already set <code>@count</code> ivar, https://github.com/mdsol/pagem/blob/accept_optional_counts_for_multiscope/lib/pagem.rb#L11, so I think this can just be made a 1 line <code>@count = @first_scope_count + @second_scope_count if @count && @count.is_a?(Integer)</code>